puma-plugin-statsd.gemspec

```diff
@@ -10,7 +10,7 @@ Gem::Specification.new do |spec|
 
   spec.files = Dir["lib/**/*.rb", "README.md", "CHANGELOG.md", "MIT-LICENSE"]
 
-  spec.add_runtime_dependency "puma", ">= 3.12", "< 5"
+  spec.add_runtime_dependency "puma", ">= 3.12", "< 6"
```

Comment:
Should this maybe be `< 5.1`?

Reply:
`< 6` seems reasonable to me, my assumption is the puma maintainers will keep 5.x highly compatible over time